lib/ex_toolkit/roman.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ExToolkit.Roman do
```

Comment:
```suggestion
defmodule ExToolkit.NumeralSystems.Roman do
```

And then we can replace `is_valid_roman/1` with just `valid?`/1, what do you think?